### PR TITLE
GridpaintActivity-tour-change to introJS

### PR DESCRIPTION
**Pull request by Ydvaaman**
hello @llaske 
i have changed the Bootstrap tour for gridpaint Activity to Intro-JS. Please review my PR.

refer to issue : https://github.com/llaske/sugarizer/issues/1090

### CODE REVIEW

File: activities/Gridpaint.activity/css/activity.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
   visibility: hidden;
 }
 
-.tutorial-title {
+/* .tutorial-title {
```
Comment (llaske) [inline activities/Gridpaint.activity/css/activity.css:31]: Do not keep commented code

File: activities/Gridpaint.activity/lib/tutorial.js [hunk 2]
```diff
@@ -2,79 +2,67 @@ define(["webL10n"], function (l10n) {
...
+			  prevLabel: l10n.get("TutoPrev"),
+			  nextLabel: l10n.get("TutoNext"),
+			  exitOnOverlayClick: false,
+			  nextToDone: false,
+			  showBullets: false,
+			})
+			.start();
+		};
+	  
+		// return tutorial/;
```
Comment (llaske) [inline activities/Gridpaint.activity/lib/tutorial.js:63]: Do not keep commented code

**Review by llaske — CHANGES_REQUESTED** [review]

File: activities/Gridpaint.activity/lib/tutorial.js (hunk 2 above)
Comment (Ydvaaman) [reply activities/Gridpaint.activity/lib/tutorial.js]: hey @llaske i have updated the code  pls check

File: activities/Gridpaint.activity/css/activity.css (hunk 1 above)
Comment (Ydvaaman) [reply activities/Gridpaint.activity/css/activity.css]: hey @llaske i have updated the code for the grid activity pls check

**Comment by llaske** [comment]
The popup doesn't appear on the right place.

image

Plus, one button is missing.

**Comment by Ydvaaman** [comment]
ok @llaske I am working on it

**Comment by Ydvaaman** [comment]
@llaske  I have fixed the it please review

**Comment by Ydvaaman** [comment]
Hey, @llaske please review this PR.

**Comment by llaske** [comment]
I thought all activities was already migrated :sweat_smile: 
Thanks @Ydvaaman

**Comment by Ydvaaman** [comment]
@llaske thanks 👍

**Pull request merged**